test/parallel/test-child-process-stdio-big-write-end.js

```diff
@@ -29,7 +30,7 @@ function parent() {
 
   // Write until the buffer fills up.
   do {
-    var buf = new Buffer(BUFSIZE);
+    buf = new Buffer(BUFSIZE);
     buf.fill('.');
     sent += BUFSIZE;
   } while (child.stdin.write(buf));
```

Comment:
@silverwind Okay, instead of introducing another variable, why don't we remove `var` from the second loop. That way, the name `buf` will be reused.